src/gm-apclientpp.cpp

```diff
@@ -615,3 +630,57 @@ double apclient_location_scouts(const char* locations_str, double create_as_hint
 
     return GM_BOOL(apclient && apclient->LocationScouts(locations_j, (int)create_as_hint));
 }
+
+double apclient_bounce(const char* data_str)
+{
+    if (api < 2)
+        return GM_FALSE;
+    const std::lock_guard<std::mutex> lock(mut);
+    json data_j;
+    try {
+        data_j = json::parse(data_str);
+    }
+    catch (std::exception ex) {
+        show_error(ex.what());
+        return GM_FALSE;
+    }
+
+    return GM_BOOL(apclient && apclient->Bounce(data_j, bounce_games, bounce_slots, bounce_tags));
+}
+
+double apclient_set_bounce_targets(const char* games_str, const char* slots_str, const char* tags_str)
+{
+    if (api < 2)
+        return GM_FALSE;
+    const std::lock_guard<std::mutex> lock(mut);
+    std::list<std::string> games_temp;
+    std::list<int> slots_temp;
+    std::list<std::string> tags_temp;
+    try {
+        games_temp = json::parse(games_str).get<std::list<std::string>>();
+        slots_temp = json::parse(slots_str).get<std::list<int>>();
+        tags_temp = json::parse(tags_str).get<std::list<std::string>>();;
```

Comment:
There is an extra semicolon
```suggestion
        tags_temp = json::parse(tags_str).get<std::list<std::string>>();
```